### PR TITLE
Fix #129: support file-version metadata (win32)

### LINKED ISSUE #129: version-string.FileVersion is not working on Windows
When I use this command

`electron-packager c:\myapp MyApp --platform=win32 --arch=x64 --version=0.32.1 --app-version=1.0.0.123 --version-string.ProductVersion=1.0.0.123 --version-string.FileVersion=1.0.0.123`

then the *File version* of MyApp.exe contains 0.32.1 instead of 1.0.0.123, see the attached screenshot.

*Windows 7 x64
electron-packager v5.1.0*

electron-packager-fileversion

### PR DESCRIPTION
**Pull request by spinico**
I had trouble cleaning up my commit logs on previous pull request. I'm not sure how to put up a test for windows executable file version though.

### CODE REVIEW

**Comment by malept** [comment]
I would add a `test/win.js`, just like there's a `test/mac.js`.

**Comment by spinico** [comment]
Added test file are required only on win32 platform. It uses native win32 functionalities to retrieve exe file metadata. For this, I needed to add a new dev dependency (rcinfo) to package.json.

File: package.json [hunk 1]
```diff
@@ -25,14 +25,15 @@
     "mv": "^2.0.3",
     "ncp": "^2.0.0",
     "plist": "^1.1.0",
-    "rcedit": "^0.3.0",
+    "rcedit": "^0.3.0",
```
Comment (malept) [inline package.json]: Please don't add trailing whitespace.

File: test/index.js [hunk 2]
```diff
@@ -20,8 +20,10 @@ series([
   require('./basic')
   require('./multitarget')
 
-  if (process.platform !== 'win32') {
-    // Perform additional tests specific to building for OS X
+  // Perform additional tests specific to OS
+  if (process.platform === 'darwin') {
```
Comment (malept) [inline test/index.js]: `process.platform === 'darwin'` is not the same as `process.platform !== 'win32'`.

File: test/index.js [hunk 3]
```diff
@@ -20,8 +20,10 @@ series([
...
     require('./mac')
+  } else if (process.platform === 'win32') {
+    require('./win32')
```
Comment (malept) [inline test/index.js]: Is it possible to run these tests via wine (particularly on Linux, so we can integrate them into the repository's continuous integration)?

File: test/index.js (hunk 2 above)
Comment (spinico) [inline test/index.js]: Ok, I'll revert that change.

File: test/index.js (hunk 3 above)
Comment (spinico) [inline test/index.js]: I'll have a look, should be possible.

File: package.json (hunk 1 above)
Comment (spinico) [inline package.json]: Sorry about that.

**Comment by spinico** [comment]
For now the CI build fails when attempting to use wine to run the win32 tests. Is there something to setup on travis for this to work?

**Comment by malept** [comment]
Yeah, wine needs to be installed first: https://github.com/malept/gmusicprocurator-app/blob/master/.travis.yml#L14

**Comment by webarchymeta** [comment]
I works for me

**Pull request merged**